View/Screens/PermissionAssignmentScreen.xaml.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Controller.DbControllers;
+using Model.Database.Contexts;
+using Model.DbModels;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+using System.Windows;
+using System.Windows.Controls;
+using View.DataContexts;
+
+namespace View.Screens
+{
+    /// <summary>
+    /// Interaction logic for PermissionAssignmentScreen.xaml
+    /// </summary>
+    public partial class PermissionAssignmentScreen : ResourceDictionary
+    {
+        private static PermissionAssignmentScreen instance;
+        public static PermissionAssignmentScreen Instance
```

Comment:
Dit mag wel het volgende worden:
`Instance => instance ??= new PermissionAssignmentScreen();`

Reply:
moet dit ook bij de PermissionAssignmentDataContext nog gedaan worden?

Reply:
Als 't goed is heb ik het voor  de rest overal gedaan, maar deze is nieuw.

Reply:
Maar ja, die mag inderdaad ook nog.